From bf5cbd06213b6fdd6dd24fb0c5dd2fe4ca3950f0 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Tue, 22 Sep 2015 14:55:54 +0200
Subject: [PATCH] setup-disk: fix detection of volume group

when we have a device like /dev/mapper/vg0-lv_root, then will the LV
name not match with lvs' first column.
---
 setup-disk.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup-disk.in b/setup-disk.in
index 55b6dcb..6d021f7 100644
--- a/setup-disk.in
+++ b/setup-disk.in
@@ -152,7 +152,7 @@ supported_boot_fs() {
 
 find_volume_group() {
 	local lv=${1##*/}
-	lvs --noheadings "$1" | awk "\$1 == \"$lv\" {print \$2}"
+	lvs --noheadings "$1" | awk '{print $2}'
 }
 
 find_pvs_in_vg() {
-- 
2.5.3

